docs/azure-saas-docs/content/en/components/Identity/identity-framework-setup-manual.md

```diff
@@ -0,0 +1,410 @@
+  - Note: An OpenSSL executable is also bundled with Git for Windows at `C:\Program Files\Git\usr\bin\openssl.exe`
+- [Visual Studio Code](https://code.visualstudio.com/)
+   - Required Extensions:
+   - [Azure AD B2C Custom Policy Extension](https://marketplace.visualstudio.com/items?itemName=AzureADB2CTools.aadb2c)
+
+## 2. Instructions
+
+### Step 1. Create B2C Tenant in Azure
+
+![(Azure Add Service Screenshot)](/azure-saas/images/identity-framework-manual-step-1-adding-service.png)
+
+- From the dashboard, click `Create a Resource` and select `Azure Active Directory B2C`
+- Note: A reference is registered in your subscription at time of creation to associate the tenant with your subscription.
+
+### Step 2. Log into newly created Tenant
+
+![(Switch Directory Screenshot)](/azure-saas/images/identity-framework-manual-step-2-switch-tenant-quicklink.png)
+
+- Click on `Open B2C Tenant` quicklink to immediately change directory to your new Tenant
+
+### Step 3. (Optional) Invite collaborators to the new Tenant
+
+Note: Collaborators are other developers you wish to help manage your services, not customers
+
+![(Tenant User Management Screenshot)](/azure-saas/images/identity-framework-manual-step-3-invite-collaborators.png)
+
+- Go to the `Azure AD B2C` dashboard while logged into your new Tenant, click `Users` in the lefthand navbar
+- After navigating, you may create or invite guest users into your Tenant allowing them access
+
+### Step 4. Create necessary app registrations in Azure AD B2C Tenant
+
+![(Registered Apps Screenshot)](/azure-saas/images/identity-framework-manual-step-4-app-registration.png)
+
+- From the Tenant `Azure AD B2C` dashboard, click `App registrations`
+- Add new registrations corresponding to the following list of modules
+- Instructions for adding client secrets, API scopes, and API permissions are detailed below
+
+#### Modules
+
+- asdk-admin-api
+  - Display Name: asdk-admin-api
+  - Account Type: Accounts in any identity provider or organizational directory
+  - Grant Admin Consent to openid and offline_access_permissions: true
+  - API Scopes:
+
+    | Scope Name | Description |
+    | - | - |
+    | tenant.read | Read a customer's own Tenant data |
+    | tenant.global.read | Admin-level read permissions for all Tenants |
+    | tenant.write | Alter a customer's own Tenant data |
+    | tenant.global.write | Admin-level write permissions for all Tenants |
+    | tenant.delete | Delete a customer's own Tenant data |
+    | tenant.global.delete | Admin-level delete permissions for all Tenants |
+
+- asdk-signupadmin-app
+  - Display Name: asdk-saas-app
+  - Account Type: Accounts in any identity provider or organizational directory
+  - Redirect URI: [Single-page application] `https://appsignup{providerName}{environmentName}.azurewebsites.net/signin-oidc`
+  - Grant Admin Consent to openid and offline_access_permissions: true
+  - Create Client Secrets?: Yes
+  - Required API Permissions: 
```

Comment:
I would do something like this:

- Required API Permissions:
   - asdk-admin-api
       - tenant.read
       - tenant.global.read
       - tenant.write
       - tenant.global.write
       - tenant.delete
       - tenant.global.delete